src/components/ProjectMembersSummary/ProjectMembersSummary.tsx

```diff
@@ -0,0 +1,37 @@
+import { Group } from '@material-ui/icons';
+import { isEmpty } from 'lodash';
+import { FC } from 'react';
+import * as React from 'react';
+import { MemberListSummary, MemberSummaryItem } from '../MemberListSummary';
+import { randomPersonPic } from '../MemberListSummary/MemberListSummary.stories';
+import { ProjectMemberListFragment } from './ProjectMembersSummary.generated';
+
+export interface ProjectMembersSummaryProps {
+  memberList?: ProjectMemberListFragment;
+}
+
+export const ProjectMembersSummary: FC<ProjectMembersSummaryProps> = ({
+  memberList,
+}) => {
+  const members: MemberSummaryItem[] | undefined = memberList?.items
+    ?.filter(({ user }) => !isEmpty(user.value?.displayFirstName?.value))
+    .map(({ user }) => ({
+      // TODO is this like card where displayFirstName is used if real first name doesn't exist
```

Comment:
Added in this fragment and helper inside api folder

Reply:
I decided it would be better/easier to have these helpers built into the API. So we actually don't need this now, we can just `firstName` or `fullName`. Sorry for the bait and switch.
https://github.com/SeedCompany/cord-api-v3/pull/504